wallet/src/constants/index.ts

```diff
@@ -3,3 +3,8 @@ export enum ChainType {
     MainNet = "MainNet",
     TestNet = "TestNet",
```

Comment:
What if we want to use local(private) network? Example is sandbox.

Reply:
another property `Private` has been added in enum to accommodate this feature.